sass/components/_global.scss

```
@@ -398,6 +398,12 @@ table {
     tr {
       border-bottom: none;
     }
+
+    > tbody {
+      > tr:nth-child(odd) {
+        background-color: $table-striped-color;
+      }
+    }
```

```suggestion
    > tbody > tr:nth-child(odd) {
      background-color: $table-striped-color;
    }
```

Since there's only one rule here, we can do this on one rule instead of nested. wdyt?